utils/build-script-impl

```diff
@@ -3147,7 +3172,9 @@ for host in "${ALL_HOSTS[@]}"; do
             # Copy executables and shared libraries from the `host_install_destdir` to
             # INSTALL_SYMROOT and run dsymutil on them.
             (cd "${CURRENT_INSTALL_DIR}" &&
-             find ./"${CURRENT_PREFIX}" -perm -0111 -type f -print | cpio --insecure -pdm "${host_symroot}")
+             find ./"${CURRENT_PREFIX}" -perm -0111 -type f -not -name "*.a" -not -name "*.py" -print | \
```

Comment:
Can you double check that we do not have any python in the sym root and any .a? My specific concern would be .py files from lldb.

Reply:
We do have indeed python files in the symbols package

```
./usr/bin/swift-api-checker.py
./System/Library/PrivateFrameworks/LLDB.framework/Versions/A/Resources/Python/lldb/utils/symbolication.py
./System/Library/PrivateFrameworks/LLDB.framework/Versions/A/Resources/Python/lldb/utils/in_call_stack.py
./System/Library/PrivateFrameworks/LLDB.framework/Versions/A/Resources/Python/lldb/macosx/crashlog.p
```

Reply:
...and also static libraries

```
./usr/lib/swift/appletvsimulator/libswiftCompatibility51.a
./usr/lib/swift/appletvsimulator/libswiftCompatibilityDynamicReplacements.a
./usr/lib/swift/appletvsimulator/libswiftCompatibility50.a
./usr/lib/swift/watchos/libswiftCompatibility51.a
./usr/lib/swift/watchos/libswiftCompatibilityDynamicReplacements.a
./usr/lib/swift/watchos/libswiftCompatibility50.a
./usr/lib/swift/iphoneos/libswiftCompatibility51.a
./usr/lib/swift/iphoneos/libswiftCompatibilityDynamicReplacements.a
./usr/lib/swift/iphoneos/libswiftCompatibility50.a
./usr/lib/swift/appletvos/libswiftCompatibility51.a
./usr/lib/swift/appletvos/libswiftCompatibilityDynamicReplacements.a
./usr/lib/swift/appletvos/libswiftCompatibility50.a
./usr/lib/swift/watchsimulator/libswiftCompatibility51.a
./usr/lib/swift/watchsimulator/libswiftCompatibilityDynamicReplacements.a
./usr/lib/swift/watchsimulator/libswiftCompatibility50.a
./usr/lib/swift/macosx/libswiftCompatibility51.a
./usr/lib/swift/macosx/libswiftCompatibilityDynamicReplacements.a
./usr/lib/swift/macosx/libswiftCompatibility50.a
./usr/lib/swift/iphonesimulator/libswiftCompatibility51.a
./usr/lib/swift/iphonesimulator/libswiftCompatibilityDynamicReplacements.a
./usr/lib/swift/iphonesimulator/libswiftCompatibility50.a
```

Reply:
In fc5fa646715af4862655da49d87fcfb7fe6b36b3 I kept this filter just before the `dsymutil` invocation -- I "folded" it into the `find` invocation so that we don't fail if no file is matched (as it could be the case with tests)

Reply:
This is also explicitly tested in `extractsymbols-default-behaviour.test` -- to support this, `cpio` is now invoked with `-v` to print all copied paths.